docs/documentation/formatting_and_styles.md

```diff
@@ -226,7 +226,7 @@ click.rich_click.STYLE_OPTIONS_TABLE_LEADING = 1
 click.rich_click.STYLE_OPTIONS_TABLE_BOX = "SIMPLE"
 ```
 
-You can make some really ~horrible~ _colourful_ solutions using these styles if you wish:
+You can make some really ~horrible~ _colorful_ solutions using these styles if you wish:
```

Comment:
Explanation of this change and the one to `editor.html`: I am implementing British English across the documents with the exception of the word "color" and its variants, which I am leaving as American English, since the code + Rich both use the word "color" and not "colour."

Another option could be: I can refer to "colour" and "colourful" to reference the general concept, but still refer to "color" as the concept inside the code.

Reply:
I'm ok with standardi[sz]ing the entirety of the docs to American English if you like.

My fingers are incapable of typing it but I don't mind, might be less confusing than having a hybrid.

I think that there are GitHub actions / pre-commit hooks that we can use to check this to catch when I inevitably trip up in the future 😁